atlasdb-feign/src/main/java/com/palantir/atlasdb/http/SerializableErrorDecoder.java

```diff
@@ -48,7 +48,9 @@ public Exception decode(String methodKey, Response response) {
 
     private static InputStream getBody(Response response) {
         try {
-            return response.body().asInputStream();
+            if (response.body() != null) {
+                return response.body().asInputStream();
+            }
         } catch (IOException e) {
             log.warn("Unable to read message body from response {}", response, e);
             return null;
```

Comment:
We'll probably need to move this return out of the catch-block if we're going ahead with this. I'm guessing currently we get an NPE that blows up, so the change makes sense in general.